bananas_server/helpers/datetime_format.py

```diff
@@ -0,0 +1,2 @@
+def datetime_to_string(datetime):
+    return datetime.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Something worth considering: for bananas-api, we use `isoformat()` to get the correct date. We do replace `+00:00` with `Z`, as that is more ISO 8601, so the end result is exactly the same. But might be nice to keep those two repos in sync style-wise.

Reply:
Yeah I initially used isoformat() and resulted in milliseconds added , so I did it this way instead.

Since I got feedback through Discord and through Github here by @LordAro I finally went with passing the datetime as an epoch UNIX timestamp, much better!